c3po/job/utils/metadata.py

```diff
@@ -60,19 +60,25 @@ def _insert_link(url, session):
         return None
 
 
-def _insert_song(track_data, session):
+def _insert_song(track_data, extras, session):
     try:
         date = datetime.strptime(track_data.year, "%Y-%m-%d")
     except ValueError:
         date = datetime.strptime(track_data.year, "%Y")
     except BaseException:
         date = None
+
+    delta = datetime.now() - extras['youtube']['posted_date']
+    factor = 24*60*60
+    score = float(extras['youtube']['views'] /(delta.days * factor))
+
     new_song = Song(
         track_data.name,
         date,
         track_data.explicit,
         track_data.popularity,
         track_data.image_id,
+        score,
```

Comment:
Use the same variable name as the column name so it is always very clear what is happening.